src/view_wrapper_test.tsx

```diff
@@ -4,10 +4,14 @@ import { mount, shallow } from 'enzyme';
 import 'mocha';
 import { always, merge, pipe } from 'ramda';
 import * as React from 'react';
-import { container, PARENT, withEnvironment } from './core';
+
+import { PARENT, withEnvironment } from './core';
 import dispatcher from './dispatcher';
 import { create as environment } from './environment';
 import Message, { Activate, Deactivate, Refresh } from './message';
+import { container } from './react';
+
+import ViewWrapper from './view_wrapper';
```

Comment:
I thank it the current implementation of `View Wrapper` will live within `@casium/react` too?

Reply:
Yup.